test/docker/docker-compose.mpush.yml

```diff
@@ -25,6 +25,8 @@ services:
       - PUSH_FCM_ENDPOINT=fcm-mock
       - TLS_SERVER_CERT_VALIDATION=false
       - FCM_AUTH_ENDPOINT=http://fcm-mock:4001
+      - PUSH_OPENAPI_EXPOSE_SPEC=false
```

Comment:
This file is used for tests on CI and local ones. Let's enable those two by default here.